agent/ibm-mq/mq_status.sh

```diff
@@ -10,6 +10,31 @@
 VERSION="1.0.0"
 echo "Version: ${VERSION}" >&2
 
+# Function to check if the user belongs to the 'mqm' group
+check_mqm_group() {
+  if ! id -nG "$USER" | grep -qw "mqm"; then
```

Comment:
Why do we need `$USER` here? Isn't that the default anyway?
Why not use `groups` for group query?
```suggestion
  if ! groups | grep -qw "mqm"; then
```